lib/rules/one-suite-per-file.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+/**
+ * @fileoverview Disallow multiple top-level suites in a single file
+ * @author Alexander Afanasyev
+ */
+
+var defaultSuiteNames = [ 'describe', 'context', 'suite' ];
+
+module.exports = function (context) {
+    var stack = [],
+        topLevelDescribes = [],
+        suiteNames = context.options[0] ? context.options[0] : defaultSuiteNames;
+
+    return {
+        CallExpression: function (node) {
+            var callee = node.callee;
+            if (callee && callee.name && suiteNames.indexOf(callee.name) > -1) {
```

Comment:
@lo1tuma yeah, I think we should support all possible suite variations. Will reuse `isDescribe`, thanks.